src/pages/styles.module.css

```diff
@@ -1,43 +1,227 @@
+}
+
+.pillPurple {
+  width: 50px;
+  height: 3px;
+  margin: 15px 0;
+  border-radius: 5px;
+  background-color: #6200ee;
+}
+
+.pillBlue {
+  width: 50px;
+  height: 3px;
+  margin: 15px 0;
+  border-radius: 5px;
+  background-color: #1ec4e9;
+}
+.pillYellow {
+  width: 50px;
+  height: 3px;
+  margin: 15px 0;
+  border-radius: 5px;
+  background-color: #f7b500;
+}
+.cardContentList {
+  list-style-type: none;
+  margin-left: -2rem;
+  font-weight: 600;
+}
+
+.link {
+  color: #bb86fc;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+.textLink {
+  color: inherit;
+  text-decoration: inherit;
+}
+
+.button {
+  outline: none;
+  width: 180px;
+  height: 50px;
+  border-radius: 4px;
+  border: none;
+  margin-top: 1rem;
+  font-size: 15px;
+  font-weight: 500;
+  letter-spacing: 1px;
+  background-color: #6200ee;
+  color: #ffffff;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: rgb(98, 0, 238, 0.8);
+}
+
+.integrationIcon {
+  width: 30px;
+  height: 30px;
+  margin: 0rem 5px;
+  border-radius: 0px;
 }
 
-.buttons {
+.startDeploying {
+  padding: 4rem;
+  background-color: #131720;
+  height: auto;
   display: flex;
+  color: #fff;
+  flex-direction: column;
   align-items: center;
+  text-align: center;
+}
+
+.infoCardSection {
+  background-color: var(--ifm-infoCard-section-color);
+  height: auto;
+  width: 100%;
+}
+
+.infoCardsIntro {
+  display: flex;
+  flex-direction: row;
   justify-content: center;
+  align-items: center;
+  flex-wrap: nowrap;
+  position: relative;
+  top: -5.5rem;
+  z-index: 99;
 }
 
-.features {
+.suggestion {
   display: flex;
+  flex-direction: row;
+  justify-content: center;
   align-items: center;
-  padding: 2rem 0;
-  width: 100%;
+  flex-wrap: nowrap;
+  position: relative;
+  top: -4rem;
+  z-index: 99;
 }
 
-.featureTitle {
-  text-align: center;
-  margin: 1rem 0;
+.suggestionText {
+  margin: 1rem 4rem;
+}
+
+.infoCardSmall {
+  width: 360px;
+  height: 160px;
+  min-height: 180px;
+  background: var(--ifm-infoCard-background-color);
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+    0px 2px 1px -1px rgba(0, 0, 0, 0.12), 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  border-radius: 2px;
+  margin: 1rem 1rem 1.2rem 1rem;
+  padding: 1.5rem;
 }
 
-.featureImage {
-  height: 100px;
-  width: 100px;
-  fill: var(--ifm-font-color-base);
+.infoCardSmall:hover {
+  box-shadow: 0px 10px 15px 0px rgba(0, 0, 0, 0.5);
+  transition: 300ms;
 }
 
-.featureAnchor svg {
-  margin-left: 5px;
-  max-width: 16px;
-  max-height: 16px;
-  fill: var(--ifm-color-primary);
-}
\ No newline at end of file
+.infoCardLarge {
+  width: 360px;
+  height: 330px;
+  min-height: 320px;
+  background: var(--ifm-infoCard-background-color);
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+    0px 2px 1px -1px rgba(0, 0, 0, 0.12), 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  border-radius: 2px;
+  margin: 0.5rem 1rem 1rem 1rem;
+  padding: 2rem 2rem 1rem 2rem;
+}
+
+@media screen and (max-width: 990px) {
```

Comment:
comment: not sure is it possible to have some breakpoints such as `xs` `sm` so we do not need to provide our own values